### PR TITLE
Load articles by id

### PR DESCRIPTION
**Pull request by wdoug**
Connects to #17

This integrates the work that @stoneyb did on viewing an article with the created articles from the `new-article` branch.

### CODE REVIEW

File: src-cljs/rmfu_ui/auth_token.cljs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(ns rmfu-ui.auth-token)
```
Comment (dviramontes) [inline src-cljs/rmfu_ui/auth_token.cljs]: probably dont need a whole namespace to encapsulate one function, but i do like the idea of having something like a utils namespace once we have enough of these functions, to keep it `DRY`

Comment (wdoug) [inline src-cljs/rmfu_ui/auth_token.cljs]: +1 on a `utils` namespace

File: src/rmfu/core.clj [hunk 2]
```diff
@@ -84,6 +84,9 @@
       (ok "Password updated!")
       (internal-server-error "Something went wrong with the password update."))))
 
+(defn get-article [id]
```
Comment (dviramontes) [inline src/rmfu/core.clj]: did you mean to leave this here ?

**Comment by dviramontes** [comment]
Other than the comment above the change-set looks good to merge.

**Pull request merged**